tests/rkt_mount_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/coreos/rkt/tests/testutils"
+)
+
+func TestMountSymlink(t *testing.T) {
+	mountSrcFile := filepath.Join(os.TempDir(), "hello")
```

Comment:
You can use `createTempDirOrPanic("rkt-mount-test-")` to get a unique temporary directory. The directory will be prefixed by "rkt" so it is obvious who created it. And several TestMountSymlink tests could run in parallel without issues.